molecule/builder/Dockerfile

```diff
@@ -6,8 +6,25 @@ ARG NAME
 LABEL org="Freedom of the Press"
 LABEL image_name="$NAME"
 
-# Build dependencies must not be added here, they are listed
-# in install_files/ansible-base/group_vars/all/securedrop
-RUN apt-get -y update && apt-get install -y python rsync sudo bash aptitude && \
-            aptitude -y dist-upgrade && \
-            apt-get clean
+# Non-version specific tooling required
+# Followed by pegged dependencies
+RUN apt-get -y update && apt-get install -y make unzip python python-pip rsync sudo bash aptitude paxctl \
```

Comment:
Is there a reason you're not pinning everything?

Reply:
i didnt think it made sense to peg some of these auxiliary tools .... buttt as with `pipefail` i dont have a good argument on why we shouldn't pin here as well.